springboot/src/main/resources/application.properties

```diff
@@ -0,0 +1,30 @@
+
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/posBD
+spring.datasource.username=postgres
+spring.datasource.password=admin
+spring.datasource.driverClassName=org.postgresql.Driver
+
+spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults = false
+
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.generate-ddl=true
+
+server.servlet.context-path=/pos
+spring.jpa.properties.hibernate.globally_quoted_identifiers=true
+
+
+# n�mero m�ximo de milissegundos que um cliente aguardar� por uma conex�o.
```

Comment:
se não tem previsão de uso, não é interessante deixar no código

Reply:
Quando usamos uma palavra-chave reservada SQL no nome de nossas entidades, o Hibernate gera instruções SQL contendo erros de sintaxe e lança SQLGrammarException , e depende do banco de dados que usamos, podemos ter este erro:
You have an error in your SQL syntax; check the manual that corresponds to your.

Ou seja, essa instrução é importante para evitar erros.

Sobre o Hikari, é algo simples que vai ajudar o sistema a evitar gargalos conforme cresce.